### DISCUSSION TITLE
Should translate term base files in the source code repository?

### DISCUSSION THREAD

**Discussion by ZhiZe-ZG** [diasurgical/DevilutionX#7201] [General]
Recently, I noticed that there are many inconsistencies in the use of terms in Chinese translation. The existing translation guide only has an English glossary, but no corresponding versions in other languages. Therefore, I want to create an English-Chinese glossary for reference by English-Chinese translators.

The existing `.po` files can mark the type of vocabulary (using the `msgctxt` section), such as "monster", "spell". But there is no type mark for potions and equipment etc. Therefore, using only `.po` as a glossary is not perfect enough.

So, should I consider further improving the structure and markup of `.po` or adding some instructions in the Wiki? Or should I maintain the English-Chinese translation glossary as another relatively independent repository?

I think this problem should also exist in translations into other languages.

**Comment by AJenbo** [comment]
Maybe it would be most easy to start with a wiki page?

msgctxt is for making it destinct from other cases where the same word is used but has a differen meaning, where are you seeing this issue with items/potions?

**Reply by ZhiZe-ZG** [reply]
There are no unhandled polysemy issues.

I mean that as a translation glossary, a term like "Rejuvenation" should be listed as a separate entry. But there is no need for a separate place for "Rejuvenation" in the game. Therefore, the `.po` files do not and should not include a separate entry for "Rejuvenation".

Perhaps, as you said, I should try to make a wiki page for this.

**Reply by AJenbo** [reply]
Maybe you would like translator comments for those strings
